python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -570,9 +571,9 @@ def main():
 
         try:
             import pysqlite2
-            pysqlite2
+            from pysqlite2 import dbapi2 as sqlite_driver
```

Comment:
why the `as sqlite_driver` part?
And if we are import dbapi2 from pysqlite2 we no longer need the previous import

Reply:
Copypasta from server repo. :-]

I was concentrating on the other things.

Reply:
np. this was a minor comment